### DISCUSSION TITLE
Good first issues and regular contributors like me

### DISCUSSION THREAD

**Discussion by doncicuto** [zitadel/zitadel#6249] [Contributors]
Hi @hifabienne  I've about a doubt about GitHub issues and our community.

My doubt is that there are some issues labeled as "good first issues" and some of them have been created some time ago. I'm tempted to assign them to me as sometimes I don't have much time to tackle bigger issues, but also I don't want to prevent new contributors to see them and try to fix'em.

Other projects that I had found earlier made impossible to me to start to help because many contributors were "predating" beginner issues so I could spent weeks trying to find anything that I could help with. Zitadel gives me all that I need, a project that uses typescript and golang, which is innovative, has a helpful team and shares my commitment with security, so I enjoy visiting the issues daily.

Now:

1. As I've spent some time with Zitadel if you prefer me to leave all those beginner issues open, please let me know. 
2. Alternatively if you feel that you're fine if I assign to me any beginner issues that hasn't been commented by newcomers for more than a month? let me know.
3. I'm having more time for the project so if you have issues that you've in the backlog but want to be fixed "soon" and may suit me, don't hesitate to assign them to me. 

Thanks a lot for your time!

**Comment by hifabienne** [comment]
Hei @doncicuto 
First to say, we are happy about every contribution, no matter if you take the ones that are labeled as "good first issues" or not.
I think it makes sense to only assign the issue, if you do have time to work on it and not to reserve them for later.
I go over the repository regularly and try to mark some issues as "good first issues", so we always have some. 
So what I want to say, I absolutely see you point, but at the moment we do not have as many regular contributers that are looking for good first issues, so I am completely fine if you like to do them. If that changes at one point I let you know. 😃

Thanks a lot for your offer about contributing also bigger issues. I will have a look at the issues and mark you when I see issues that would be great if you could contribute.

**Comment by doncicuto** [comment]
Thanks a lot @hifabienne, I appreciate these guidelines

... all those issues are mine now muahahahaha :smile:
